spec/features/user_spec.rb

```diff
@@ -79,6 +79,6 @@
     click_link 'Выйти'
     visit sign_up_path
     register_user
-    expect(page).to have_content('has already been taken')
+    expect(page).to have_content('Данный E-mail уже зарегистрирован в системе')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.